.eslintrc.js

```diff
@@ -1,5 +1,29 @@
 module.exports = {
+    parser: '@typescript-eslint/parser',
     extends: [
         '@jitsi/eslint-config'
-    ]
+    ],
+    plugins: [
+        '@typescript-eslint'
+    ],
+    rules: {
```

Comment:
I think it's simpler and avoids having to depend directly on the parser if you use an override like so: https://github.com/jitsi/jitsi-meet/blob/f060ab9d26478dc3fe49f17e38a7994225a295fc/react/.eslintrc.js#L11

Reply:
done